### PR TITLE
Bug 1727299 - Document new text metric

### PR DESCRIPTION
**Pull request by badboy**
[doc only]

### CODE REVIEW

File: docs/user/reference/metrics/text.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+<div data-lang="Java" class="tab"></div>
+
+<div data-lang="Swift" class="tab"></div>
+
+<div data-lang="Python" class="tab"></div>
+
+<div data-lang="JavaScript" class="tab">
+
+```js
+import * as display from "./path/to/generated/files/display.js";
```
Comment (brizental) [inline docs/user/reference/metrics/text.md:31]: This example is a bit confusing as `display` is imported but the name of the metric used is `article`.

File: docs/user/reference/metrics/text.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+assert.strictEqual("some content", await article.content.testGetValue());
```
Comment (brizental) [inline docs/user/reference/metrics/text.md:76]: Some comment about the confusing import.

File: docs/user/reference/metrics/text.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+<div data-lang="Rust" class="tab"></div>
+
+<div data-lang="Firefox Desktop" class="tab"></div>
+
+{{#include ../../../shared/tab_footer.md}}
+
+#### Limits
+
+* Text metrics can only be sent in custom pings.
```
Comment (brizental) [inline docs/user/reference/metrics/text.md:45]: Please linkinfy "custom pings".

File: docs/user/reference/metrics/text.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+* Text metrics are always of data collection category 3 (`web_activity`) or category 4 (`highly_sensitive`).
+* Only `ping` and `application` lifetime are allowed.
```
Comment (brizental) [inline docs/user/reference/metrics/text.md:47]:
```suggestion
* Only `ping` and `application` lifetimes are allowed.
```

File: docs/user/reference/metrics/text.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
+# Text
+
+Records a single long Unicode text, used when the limits on `String` are too low.
+
+{{#include ../../../shared/blockquote-warning.html}}
+
+## Important
+
+> This type should only be used in special cases. See [limitations](#limits) below.
+> It is best to talk to the Glean team before using this.
```
Comment (brizental) [inline docs/user/reference/metrics/text.md:10]: Kinda weird for me that we suggest people ask us before using a specific metric type in their own apps.

Comment (badboy) [reply docs/user/reference/metrics/text.md:10]: Yeah, I wasn't sure about this. I'm fine with removing that sentence, the limitations should be enough of a "warning"

**Pull request merged**